### PR TITLE
Feature/add query param

### PR DESCRIPTION
**Pull request by yamyn**
Hi)
I use this simple and at the same time useful module constantly, but I constantly lack the functionality to introduce filtering and sorting methods, could you accept a small change that does not affect your product and improves functionality.

### CODE REVIEW

**Comment by dkatavic** [comment]
Hi @yamyn . Sorry for a very late reply, I missed the notification.

I like the changes. Can you bump the minor version in `package.json` to make sure that deploy pipeline publishes the new package?

**Comment by yamyn** [comment]
@dkatavic, Yes, sure.

**Comment by yamyn** [comment]
@dkatavic, Done)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "onedrive-api",
-  "version": "0.3.4",
+  "version": "0.4.5",
```
Comment (dkatavic) [inline package.json:3]: This should be `"0.4.0"`, after updating minor version we are resetting patch version

Comment (yamyn) [reply package.json:3]: Understood

**Review by dkatavic — APPROVED** [review]
Looks great, you can merge the changes. Thanks for the contribution :)

**Comment by dkatavic** [comment]
@yamyn Thanks for the contribution, I have merged the changes

**Pull request merged**